presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1978,10 +1980,25 @@ private AccumulatorFactory buildAccumulatorFactory(
                 maskChannel = aggregation.getMask().map(value -> source.getLayout().get(value));
             }
 
+            List<SortOrder> sortOrders = ImmutableList.of();
+            List<Symbol> sortKeys = ImmutableList.of();
+            if (aggregation.getCall().getOrderBy().isPresent()) {
+                OrderBy orderBy = aggregation.getCall().getOrderBy().get();
+
+                sortKeys = orderBy.getSortItems().stream()
+                        .map(SortItem::getSortKey)
+                        .map(Symbol::from)
+                        .collect(toImmutableList());
+
+                sortOrders = orderBy.getSortItems().stream()
+                        .map(QueryPlanner::toSortOrder)
+                        .collect(toImmutableList());
```

Comment:
Maybe these two above section could be moved to `OrderBy` class so here you would just call:
```
sortKeys = orderBy.getSortKeys();
sortOrders = orderBy.getSortOrders();
```
What do you think?

Reply:
I did that when I started this refactoring, but then I realized it's only used here, so I decided to inline it. 

Also, OrderBy is an AST class, but SortOrder is not. So, I would still need to do some transformation here.